qt_dotgraph/package.xml

```diff
@@ -13,10 +13,10 @@
   <url type="bugtracker">https://github.com/ros-visualization/qt_gui_core/issues</url>
   <author>Thibault Kruse</author>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
 
-  <run_depend>python-pydot</run_depend>
-  <run_depend version_gte="0.3.0">python_qt_binding</run_depend>
+  <exec_depend>python-pydot</exec_depend>
+  <exec_depend version_gte="0.3.0">python_qt_binding</exec_depend>
```

Comment:
Both `python-` keys probably need to be changed to `python3-...` and corresponding rosdep keys created in rosdistro (if they don't exist already).

Reply:
They actually do exist. I forgot to change this line when I refactored PRs. I'll update this